src/main/java/model/EventModel.java

```diff
@@ -112,18 +113,18 @@ public boolean resolveParticipant(ParticipantBean participant) {
 		if(participant.getEventId() != 0 && participant.getUserId() != 0) {
 			int eventId = participant.getEventId();
 			int userId = participant.getUserId();
-			switch (ActionEnum.fromInt(participant.isAction())) {
+			switch (participant.getAction()) {
 				case ENTER_EVENT:
-					result = daoFactory.getEventDao().joinEvent(eventId, userId, 1); 
+					result = daoFactory.getEventDao().joinEvent(eventId, userId, StatusEnum.getValue(StatusEnum.APPROVED));
 					break;
```

Comment:
This is something that should be changed later on as it is not a proper way to implement an enum, for now it will be approved as it is functional. Proper way to do this would be to simply call StatusEnum.APPROVED.getStatus() instead of using the getValue method which is a method that is used to find a value in enum class if it even exists there, since in this case you already know for certain that the enumerated value exists and you simply need to get the number associated with the constant there is no need to traverse the entire list. Currently you could look at this as if you know there is an array of values such as [1,2,3] and you are certain that array[1] contains a value which is = 2 but you will not  fetch that value by going directly to its position instead the thing that is done here is iteration through entire array to find a value for which you already know the position of. It works but there are redundant operations made.